sql/core/src/test/resources/sql-tests/inputs/describe-table-column.sql

```diff
@@ -34,6 +34,19 @@ DESC FORMATTED desc_complex_col_table col;
 -- Describe a nested column
 DESC FORMATTED desc_complex_col_table col.x;
 
+-- Test output for histogram statistics
+SET spark.sql.statistics.histogram.enabled=true;
+SET spark.sql.statistics.histogram.numBins=2;
+
+INSERT INTO desc_col_table values 1, 2, 3, 4;
+
+ANALYZE TABLE desc_col_table COMPUTE STATISTICS FOR COLUMNS key;
+
+DESC EXTENDED desc_col_table key;
+
+SET spark.sql.statistics.histogram.enabled=false;
+SET spark.sql.statistics.histogram.numBins=256;
```

Comment:
We don't need to reset the config, the test framework will reset the entire session after each test.

Reply:
isn't it better to reset them to prevent possible future tests eventually added after these lines to be influenced? If no, I can remove them.

Reply:
You can also argue that this increase the maintenance cost as we need to change it if we change the default value of these configs. I think the general principle is, set the config if the following queries depend on it. At the end of test, there is no following queries and we don't need to do anything to the configs.